src/fiskaltrust.Launcher/Commands/Common.cs

```diff
@@ -24,13 +24,17 @@ public CommonCommand(string name) : base(name)
             AddOption(new Option<string?>("--log-folder"));
             AddOption(new Option<LogLevel?>("--log-level"));
             AddOption(new Option<string>("--launcher-configuration-file", getDefaultValue: () => "launcher.configuration.json"));
+            AddOption(new Option<string>("--legacy-config-file", getDefaultValue: () => "fiskaltrust.exe.config"));
+            AddOption(new Option<bool>("--use-legacy-config", getDefaultValue: () => false));
```

Comment:
And if the default value `launcher.configuration.json` is used and the file is not found we check if a `fiskaltrust.exe.config` exists and create a `launcher.configuration.json` with the migrated content.
Then we rename the `fiskaltrust.exe.config` to `fiskaltrust.exe.config.migrated`.
And maybe print a warning that we did that.

But that part is maybe to intransparent 🤷

Reply:
Or maybe even If a `fiskaltrust.exe.config` is found in the default path it is read and the `launcher.configuration.json` file is merged into it 
🤔

Reply:
And maybe call the parameter `--merge-legacy-config-if-exists`